pydeseq2/utils.py

```diff
@@ -675,6 +683,7 @@ def trimmed_variance(x, trim=0.125, axis=1):
     return 1.51 * trimmed_mean(sqerror, trim=trim, axis=axis)
 
 
+# TODO : bug here in multi-factor?
```

Comment:
can you give more info for the future dev trying to fix it? when exactly does it happen, what kind of bug?

Reply:
you can also add it to the list of issues after merging with exact explanation, with the code snippet on how to get it and what you expect instead

Reply:
This is a note I left for myself which I meant to delete later on. Thanks for catching it!